Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN echo "sha-${SHA}" > /etc/school-experience-sha
 
 # hadolint ignore=DL3018
 RUN apk add --no-cache bash build-base git tzdata libxml2 libxml2-dev \
-			postgresql-libs postgresql-dev nodejs yarn
+			postgresql-libs postgresql-dev nodejs yarn chromium chromium-chromedriver
```

Comment:
There are small chances the chromium-chromedriver version falls behind the chromium version which will cause the tests to fail and block our releases. 

I've faced that issue only once before, so I'm OK with this change. If it happens again we can either refactor it to always download the same versions or disable the feature tests in our CI/CD for few days until the compatible version is released.

Reply:
Ah that's a good point; I'll maybe pin these versions so that it doesn't happen unexpectedly 👍